packages/sfui/shared/slider/slider.ts

```diff
@@ -71,9 +71,11 @@ export default class VSFSlider {
       const rect = children[index].getBoundingClientRect();
       const left = rect.left - container.getBoundingClientRect().left;
       const top = rect.top - container.getBoundingClientRect().top;
+      const centerHorizontal = container.clientWidth / 2 - rect.width / 2;
+      const centerVertical = container.clientHeight / 2 - rect.width / 2;
```

Comment:
```suggestion
      const centerVertical = container.clientHeight / 2 - rect.height / 2;
```
Should be height here?

Reply:
Ok, changed it.